<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detalle_publicacion</class>
 <widget class="QWidget" name="detalle_publicacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>10</y>
     <width>661</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pregunta</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_usuario">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>841</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Medium</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_titulo">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>1291</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: white;
	border-width: 2px;
border-style: solid;
	border-color: gray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_descripcion">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>360</y>
     <width>1291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: white;
	border-width: 2px;
border-style: solid;
	border-color: gray;
color: gray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_foto">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	border-width: 2px;
border-style: solid;
	border-color: gray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_comentario">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>480</y>
     <width>1291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color: white;
	border-width: 2px;
border-style: solid;
	border-color: gray;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="placeholderText">
    <string>Escribe una respuesta...</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabla_comentarios">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>580</y>
     <width>1291</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget{
	background-color: white;
	border-width: 2px;
border-style: solid;
	border-color: gray;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_EnviarComentario">
   <property name="geometry">
    <rect>
     <x>1570</x>
     <y>500</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #086FE6;
color: white;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>Enviar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_comentarios">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Medium</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Comentarios</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_IrPerfil">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #086FE6;
color: white;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>Ir perfil</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_like">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #086FE6;
color: white;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>Like</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_recomendar">
   <property name="geometry">
    <rect>
     <x>1420</x>
     <y>100</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #086FE6;
color: white;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>Recomendar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
